app/views/content_items/detailed_guide.html.erb

```diff
@@ -1,45 +1,49 @@
 <div class="grid-row">
   <div class="column-two-thirds">
-    <%= render 'govuk_component/title', @content_item.title_and_context %>
+    <%= render 'govuk_component/title',
+       title: @content_item.title,
+       average_title_length: "long" %>
```

Comment:
No context is being passed here, so the title is missing the grey "Guidance" text above the title.